tests/issues/issue-2475.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+return [
+    'issue-2370-A' => [
```

Comment:
Please rename this test to "issue-2475-A"

Reply:
done